src/game/common/rts/team.cpp

```diff
@@ -0,0 +1,817 @@
+    str = d->Get_AsciiString(g_teamUnitType3Key, &exists);
+
+    if (max > 0 && exists) {
+        m_unitsInfo[m_numUnitsInfo].min_units = min;
+        m_unitsInfo[m_numUnitsInfo].max_units = max;
+        m_unitsInfo[m_numUnitsInfo].unit_thing_name = str;
+        m_numUnitsInfo++;
+    }
+
+    min = d->Get_Int(g_teamUnitMinCount4Key, &exists);
+    max = d->Get_Int(g_teamUnitMaxCount4Key, &exists);
+    str = d->Get_AsciiString(g_teamUnitType4Key, &exists);
+
+    if (max > 0 && exists) {
+        m_unitsInfo[m_numUnitsInfo].min_units = min;
+        m_unitsInfo[m_numUnitsInfo].max_units = max;
+        m_unitsInfo[m_numUnitsInfo].unit_thing_name = str;
+        m_numUnitsInfo++;
+    }
+
+    min = d->Get_Int(g_teamUnitMinCount5Key, &exists);
+    max = d->Get_Int(g_teamUnitMaxCount5Key, &exists);
+    str = d->Get_AsciiString(g_teamUnitType5Key, &exists);
+
+    if (max > 0 && exists) {
+        m_unitsInfo[m_numUnitsInfo].min_units = min;
+        m_unitsInfo[m_numUnitsInfo].max_units = max;
+        m_unitsInfo[m_numUnitsInfo].unit_thing_name = str;
+        m_numUnitsInfo++;
+    }
+
+    min = d->Get_Int(g_teamUnitMinCount6Key, &exists);
+    max = d->Get_Int(g_teamUnitMaxCount6Key, &exists);
+    str = d->Get_AsciiString(g_teamUnitType6Key, &exists);
+
+    if (max > 0 && exists) {
+        m_unitsInfo[m_numUnitsInfo].min_units = min;
+        m_unitsInfo[m_numUnitsInfo].max_units = max;
+        m_unitsInfo[m_numUnitsInfo].unit_thing_name = str;
+        m_numUnitsInfo++;
+    }
+
+    min = d->Get_Int(g_teamUnitMinCount7Key, &exists);
+    max = d->Get_Int(g_teamUnitMaxCount7Key, &exists);
+    str = d->Get_AsciiString(g_teamUnitType7Key, &exists);
+
+    if (max > 0 && exists) {
+        m_unitsInfo[m_numUnitsInfo].min_units = min;
+        m_unitsInfo[m_numUnitsInfo].max_units = max;
+        m_unitsInfo[m_numUnitsInfo].unit_thing_name = str;
+        m_numUnitsInfo++;
+    }
+
+    str = d->Get_AsciiString(g_teamHomeKey, &exists);
+
+    m_homeLocation.x = 0.0f;
+    m_homeLocation.y = 0.0f;
+    m_homeLocation.z = 0.0f;
+    m_hasHomeLocation = false;
+
+    if (exists) {
+        for (const Waypoint *i = g_theTerrainLogic->Get_First_Waypoint(); i != nullptr; i = i->Get_Next()) {
+            if (i->Get_Name() == str) {
+                m_homeLocation = *i->Get_Location();
+                m_hasHomeLocation = true;
+            }
+        }
+    }
+
+    m_scriptOnCreate = d->Get_AsciiString(g_teamOnCreateScriptKey, &exists);
+    m_isAIRecruitable = d->Get_Bool(g_teamIsAIRecruitableKey, &exists);
+
+    if (!exists) {
+        m_isAIRecruitable = false;
+    }
+
+    m_isBaseDefense = d->Get_Bool(g_teamIsBaseDefenseKey, &exists);
+    m_isPerimeterDefense = d->Get_Bool(g_teamIsPerimeterDefenseKey, &exists);
+    m_automaticallyReinforce = d->Get_Bool(g_teamAutoReinforceKey, &exists);
+    int aggressiveness = d->Get_Bool(g_teamAggressivenessKey, &exists);
+    m_aggressiveness = 0;
```

Comment:
This can go into `else` of condition below.

Reply:
This is what original looks like and it should be fine.